glusterd2/commands/snapshot/snapshot-create.go

```diff
@@ -591,11 +597,9 @@ func duplicateVolinfo(vol, v *volume.Volinfo) {
 	 */
 	return
 }
-func snapshotBrickCreate(snapVolinfo, volinfo *volume.Volinfo, brickinfo *brick.Brickinfo, brickCount int) string {
-	mountData := brickinfo.MountInfo
-	nodeID := strings.Replace(brickinfo.ID.String(), "-", "", -1)
-	SnapDirPrefix := config.GetString("rundir") + "/snaps/"
-	brickPath := fmt.Sprintf("%s%s/%s/%s/brick%d%s", SnapDirPrefix, volinfo.Name, nodeID, snapVolinfo.Name, brickCount, mountData.Mountdir)
+func snapshotBrickCreate(snapName, volName, nodeID, mountDir string, brickCount int) string {
```

Comment:
Needs an empty line above. Also rename to `snapshotBrickPath` or something else that describes this function better.